static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -498,7 +498,7 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
 
       const entityId = cardId.replace('js-yl-', '');
```

Comment:
I think we'll want to rename this from entityId, entityId has a specific meaning inside the knowledge graph. For instance if you go to a specific entity the url will look like `https://www.yext.com/s/<business id>/entity/edit3?entityIds=<entityId>`

Reply:
that URL `<entityId>` is actually the same as this `uid`. there are 2 values that are referred to as "entityId". the internal one is the one included in the entity url and is returned in the response data as `uid`. the external one is the one the user can change when editing an entity and is returned as the `id`

Reply:
ah ok so it was actually "kind of" misnamed before?

Reply:
yes, it was a little misleading. in alpha, the `uid` is the `entityId` and the `id` is `externalId`